simapp/state.go

```diff
@@ -94,7 +94,7 @@ func AppStateRandomizedFn(
 		numInitiallyBonded = numAccs
 	}
 
-	fmt.Printf(
+	log.Printf(
```

Comment:
I'm not totally sure this is right, is this what the SDK simulation uses, or is it like simManager.ctx.log

Reply:
I dont think they have a seperate logger for simulation